boards/bluepill/Makefile.include

```diff
@@ -15,16 +15,18 @@ include $(RIOTMAKE)/tools/serial.inc.mk
 # To stop bootloader from loading an existing firmware, pull down
 # (ground) GPIO B1.
 ifeq ($(PROGRAMMER),dfu-util)
-	export LINKER_SCRIPT = stm32f103c8_bluepill.ld
-	export BINFILE = $(patsubst %.elf,%.bin,$(ELFFILE))
+  export BINFILE = $(patsubst %.elf,%.bin,$(ELFFILE))
```

Comment:
This line defines BINFILE, but L24 below uses HEXFILE. Is there a use for this variable?

Reply:
Well, actually I just moved the indentation since there were tabs instead of 2 spaces, which is the convention. I think whatever modification of the code itself should be in another PR.